test/unit/specs/util/EntityToStateMapper.spec.js

```diff
@@ -49,6 +49,25 @@ describe('Entity to state mapper', () => {
       expect(field.visible()).to.equal(true)
     })
 
+    it('should have working expressions', () => {
+      const compoundString = field.children[2]
```

Comment:
should wrap current it into a describe and have separate more describing it's for each case, see hellper test as example